starknet_py/hash/utils.py

```diff
@@ -3,11 +3,9 @@
 
 from crypto_cpp_py.cpp_bindings import ECSignature, cpp_hash
 from eth_utils.crypto import keccak
-from starkware.cairo.lang.vm.crypto import pedersen_hash as default_hash
 from starkware.crypto.signature.signature import sign
 
 from starknet_py.common import int_from_bytes
-from starknet_py.utils.crypto.facade import use_cpp_variant
```

Comment:
If we decide to remove usages of `default_hash` we should do the following:
- modify the docs (I remember there is something about disabling cpp extension)
- remove `use_cpp_variant`